pkg/storage/config/config.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"os"
+)
+
+type Config struct {
+	StoreType     string
+	TokenFilePath string
+}
+
+const (
+	k8s  = "k8s_secrets"
+	none = "none"
+)
+
+// should we check here what the ENV says and act accordingly with values?
+func NewFromEnv() (*Config, error) {
```

Comment:
because we want to keep it reuseable. This NewFromEnv will see both flows- for k8s and none. For k8s, the initialization would `storageConfig.NewFromEnv()` and not storageConfig.NewFromEnv(""). We could definitely demand that a param is passed in (`""` or `"/run/conjur/access-token"`). Let me know what you think.